src/App/Entity/Talk.php

```diff
@@ -77,57 +77,58 @@ public static function fromStandardTalk(
         DateTimeImmutable $time,
         Speaker $speaker,
         string $title,
-        string $abstract
+        string $abstract,
```

Comment:
I agree that it is easier to just check if it's null or not. But we require an abstract right? So it should never be null?

I think we're going in circles :stuck_out_tongue: 

Could you assert that you don't have an empty string?